.github/workflows/container-image.yml

```diff
@@ -250,6 +261,11 @@ jobs:
         with:
           image: "samba-server:default-fedora-amd64"
           container_engine: ${{ env.CONTAINER_CMD }}
+      - name: Fetch server default-fedora-arm64
+        uses: ishworkh/container-image-artifact-download@v2.0.0
+        with:
+          image: "samba-server:default-fedora-arm64"
+          container_engine: ${{ env.CONTAINER_CMD }}
```

Comment:
I'm going to recommend skipping the tests for now. Let's just get the builds on emulated arm64 in this initial PR.